packages/tooltips/stories/2-Tooltip.stories.tsx

```diff
@@ -0,0 +1,180 @@
+    | 'bottom-start'
+    | 'bottom-end'
+    | 'end'
+    | 'end-top'
+    | 'end-bottom'
+    | 'start'
+    | 'start-top'
+    | 'start-bottom';
+  size: TOOLTIP_SIZE;
+  hasArrow: boolean;
+  forceVisibility: boolean;
+  delayMS: number;
+}
+
+export const Default: Story<IStoryProps> = ({
+  placement,
+  size,
+  hasArrow,
+  forceVisibility,
+  delayMS
+}) => {
+  return (
+    <Grid>
+      <Row alignItems="center" style={{ minHeight: 500 }}>
+        <Col md={6} style={{ textAlign: 'center' }}>
+          <Tooltip
+            content={
+              <>
+                <Title>Dark Tooltip</Title>
+                <Paragraph>{retrieveTooltipContent(size, 'dark')}</Paragraph>
+              </>
+            }
+            hasArrow={hasArrow}
+            placement={placement}
+            size={size === 'default' ? undefined : size}
+            delayMS={delayMS}
+            isVisible={forceVisibility ? true : undefined}
+            zIndex={1}
+            popperModifiers={{ preventOverflow: { boundariesElement: 'viewport' } }}
+          >
+            <Button>Default tooltip</Button>
+          </Tooltip>
+        </Col>
+        <Col md={6} style={{ textAlign: 'center' }}>
+          <Tooltip
+            content={
+              <>
+                <Title>Light Tooltip</Title>
+                <Paragraph>{retrieveTooltipContent(size, 'light')}</Paragraph>
+              </>
+            }
+            hasArrow={hasArrow}
+            placement={placement}
+            size={size === 'default' ? undefined : size}
+            type="light"
+            isVisible={forceVisibility ? true : undefined}
+            delayMS={delayMS}
+            zIndex={1}
+            popperModifiers={{ preventOverflow: { boundariesElement: 'viewport' } }}
+          >
+            <Button>Light tooltip</Button>
+          </Tooltip>
+        </Col>
+      </Row>
+      <Row>
+        <Col>
+          {size !== 'default' && (
+            <Alert type="warning">
+              <AlertTitle>Size warning</AlertTitle>
+              Although small, light tooltips and large, dark tooltips are supported in code, they go
+              against Garden&apos;s usage guidelines. Please don&apos;t use this.
+            </Alert>
+          )}
+        </Col>
+      </Row>
+    </Grid>
+  );
+};
+
+Default.argTypes = {
+  placement: {
+    name: 'Placement',
+    control: {
+      type: 'select',
+      options: [
+        'auto',
+        'top',
+        'top-start',
+        'top-end',
+        'bottom',
+        'bottom-start',
+        'bottom-end',
+        'end',
+        'end-top',
+        'end-bottom',
+        'start',
+        'start-top',
+        'start-bottom'
+      ]
+    }
+  },
+  hasArrow: {
+    name: 'Has Arrow'
+  },
+  size: {
+    name: 'Size',
+    control: {
+      type: 'select',
+      options: ['default', 'small', 'medium', 'large', 'extra-large']
+    }
+  },
+  forceVisibility: {
+    name: 'Foce Visibility'
+  },
+  delayMS: {
```

Comment:
If controls can be dynamically disabled, then `delayMS` should be disabled when `forceVisibility` is true. https://storybook.js.org/docs/react/essentials/controls#disable-controls-for-specific-properties

Reply:
I don't think we have access to the current story state in the `argTypes`. It's static like `propTypes`.

Another option is a warning message within the story based on those states, but that might be heavy-handed.

Reply:
Yeah, no need to get fancy with a warning.